src/App.jsx

```diff
@@ -23,10 +23,7 @@ function App() {
         <NewFirstScreen />
```

Comment:
Cool that you use conditional rendering here 😎.

Reply:
Thank you 😊